### DISCUSSION TITLE
Extension host terminated unexpectedly

### DISCUSSION THREAD

**Discussion by Robert-Springer** [community/community#11907] [Codespaces]
When I open VS Code I receive a message "Extension host terminated unexpectedly"

![image](https://user-images.githubusercontent.com/37991279/155149050-63e38bb0-a664-4f53-840a-ba7417819ee4.png)

**Comment by joeSaad** [comment]
facing the same issue and posted a question about it here: https://github.com/microsoft/vscode/issues/143623

Completely blocked now, cannot use VSCode

**Reply by Robert-Springer** [reply]
Thank w Joe, I'm following the thread as well.
